### ISSUE TITLE
Collaboration Guide Documentation

### ISSUE THREAD

**Issue by dinaolmelak** [Techwise-Team-15/my_pet#24]
# CollabGuide.md

### Description
If we would want to progress faster, we would need to have a faster way of working on tickets without waiting on the completion of another ticket. Hence, having this guide will help cement how we can collaborate with one another and get more tickets done.

### Documentation Area
This Documentation goes over the steps to follow when **working on ticket**

### Documentation Objective
This documentation should have a clear set of steps to follow for a team member to effectively collab.

### Documentation Content
Outline the key points or sections to be covered in the documentation. This can include:

- [ ] Overview of the feature/component
- [ ] Installation instructions
- [x] Usage guidelines and examples
- [ ] API documentation (if applicable)
- [ ] Configuration options
- [ ] Troubleshooting Tips

### Additional Context
N/A

### Acceptance Criteria
Specify the criteria that must be met for this documentation task to be considered complete. For example:

- [x] All key points and sections are covered in the documentation.
- [x] The documentation is clear, concise, and well-structured.
- [x] Screenshots or diagrams are included where necessary.
